### PR TITLE
OHRM-1083:Initiate migrate unit tests to phpunit 9

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,30 +1,23 @@
...
+      <directory>symfony/config</directory>
+      <directory>symfony/data</directory>
+      <directory>symfony/lib</directory>
+      <directory>symfony/web</directory>
+    </exclude>
+  </coverage>
+  <testsuites>
+    <testsuite name="All Tests">
+      <!--            <file>symfony/test/PluginAllTests.php</file>-->
+      <file>symfony/plugins/orangehrmAdminPlugin/test/model/service/JobTitleServiceTest.php</file>
```
Comment (samanthajayasinghe) [inline phpunit.xml:19]: That may work with 
```
*/test/*Test.php
```

File: symfony/plugins/orangehrmAttendancePlugin/client/components/HelloWorld.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<template>
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmAttendancePlugin/client/components/HelloWorld.vue:1]: Why Vue fiiles?

File: phpunit.xml (hunk 1 above)
Comment (RajithaKumara) [reply phpunit.xml:19]: `symfony/test/PluginAllTests.php` also working. But the initial point it's breaking all unit tests. Once finish with the admin module we can replace these files with `symfony/plugins/orangehrmAdminPlugin/test/orangehrmAdminPluginAllTests.php` once done with all modules we can replace `symfony/test/PluginAllTests.php`.

**Comment by RajithaKumara** [comment]
Related: #647

**Review by samanthajayasinghe — APPROVED** [review]

**Pull request merged**